modules/drivers/snowflake/resources/metabase-plugin.yaml

```diff
@@ -40,6 +40,7 @@ driver:
     - merge:
       - additional-options
       - placeholder: 'queryTimeout=0'
+    - alternative-hostname
```

Comment:
Since this is only used in one place you shouldn't define it in `driver.common`... just define the whole thing inline like https://github.com/metabase/metabase/blob/master/modules/drivers/snowflake/resources/metabase-plugin.yaml#L20-L24